### ISSUE TITLE
How to select city, state or zip code?  By noaa query of locations search?  Then map to alert?

### ISSUE THREAD

**Issue by dachshund-digital** [Dolphman/siren#1]
How to select city, state or zip code?  By noaa query of locations search?  Then map to alert?  Asking here, since I did not see anyway to ask you a question directly?

**Comment by DolphDev** [comment]
This module is no longer supported and pretty buggy. I believe there are other libraries to wrap around the same service much better than this one.

**Comment by dachshund-digital** [comment]
Yes, I found some alternate methods.  But thank for the reply!  Most 
appreciated.

Schor-
